### PR TITLE
Add channel renaming feature

### PR DESCRIPTION
**Pull request by pwalczysko**
Resulting from the comment in https://github.com/ome/omero-guide-iviewer/pull/5#pullrequestreview-457337033

The channel renaming feature belongs to the Preview and thus to this guide, rather than iviewer.

cc @manics @jburel

### CODE REVIEW

File: docs/data-management.rst [hunk 1]
```diff
@@ -134,6 +134,13 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+#. Stay in ``General`` tab of the right-nand pane and adjust the channel names:
+
+   - Select any image inside that Dataset and click on the pencil |image16| icon in the right-hand pane next to Channels.
+
+   - Input “DAPI” instead of channel “457” and “GFP”, “Aurora-B” and “CY-6\ ” for the other channels.
```
Comment (mtbc) [inline docs/data-management.rst]: That last channel name's a bit weird, it needs that final space?

**Review by manics — COMMENTED** [review]
Other than Mark's comment this looks fine

File: docs/data-management.rst (hunk 1 above)
Comment (pwalczysko) [reply docs/data-management.rst]: @mtbc no, this is an artifact from text format conversions. Will fix, thanks

Comment (pwalczysko) [reply docs/data-management.rst]: Fiexed in https://github.com/ome/omero-guide-introduction/pull/9/commits/e1b20c7107164f330ea947d65c7a507b0c0d84af

**Comment by pwalczysko** [comment]
Thank you @manics  @mtbc 

Merging

**Pull request merged**